pkg/clickhouse/config.go

```diff
@@ -28,10 +29,18 @@ func (c *Config) ToClickHouseOptions() *click_house.Options {
 }
 
 func (c *Config) GetIngestrURI() string {
+	//nolint:nosprintfhostport
+	uri := fmt.Sprintf("clickhouse://%s:%s@%s:%d", c.Username, c.Password, c.Host, c.Port)
```

Comment:
I would recommend using `url` package to build the URI. It will automatically escape URI characters, making this a lot more robust.

Reply:
Here's an example:
https://github.com/bruin-data/bruin/blob/main/pkg%2Fgoogleads%2Fconfig.go#L16